extensions/database/module/langs/translation-en.json

```
@@ -47,6 +47,7 @@
     "database-parsing/worksheet": "Worksheets",
     "database-parsing/option": "Options",
     "database-parsing/preview-button": "Update&nbsp;Preview",
```

This is not strictly speaking related to your changes, but this button seems to have escaped our normalization of casing (#3363), so let me suggest this change anyway:
```suggestion
    "database-parsing/preview-button": "Update&nbsp;preview",
```

---

@johnvl2 Hello and sorry for the late reply
One thing I have noticed is that the button "Ignore first" has no effect alone, you must also enter a number of rows to ignore.
But yes, to have a real test, you have no other choice than to "wait", because the dom is not rendered, and that's maybe something you can add as a comment next to it
Also, you can probably keep only one single assertion, about one single cell of the grid to improve the test readbility

Thx,

---

Updated to adhere to casing standards.